tests/network/vmi_passt.go

```diff
@@ -213,7 +213,6 @@ EOL`, inetSuffix, serverIP, serverPort)
 						Expect(err).ToNot(HaveOccurred())
 						clientVMI = tests.WaitForSuccessfulVMIStartIgnoreWarnings(clientVMI)
 						Expect(console.LoginToAlpine(clientVMI)).To(Succeed())
-						console.LoginToAlpine(clientVMI)
```

Comment:
This looks like an accidental removal, can that be?

Reply:
I don't think so - this removed line is 100% equal to the one above (except that its error is not checked).

Reply:
Oh, right. :+1: